medical_emr/__openerp__.py

```diff
@@ -36,57 +37,10 @@
 
     'name': 'Medical EMR : Electronic Medical Record management for Medical',
     'version': '1.0',
-    'author': "Medical Team,Odoo Community Association (OCA)",
+    'author': "Medical Team, LasLabs, Odoo Community Association (OCA)",
```

Comment:
To keep consistency with master medical module, I propose: **Odoo** Medical Team, ...